### PR TITLE
Feature/search button

### PR DESCRIPTION
**Pull request by Denzel-Turambi**
**What I did**:
- Added search button with functionality and styling

**HTML**:
 - wrapped search elements in a div (using form breaks the website)

**domUpdates.js**:
-  query selector and event listener for search button, and added conditionals to renderFiltedRecipes and renderSavedFilteredRecipes

**CSS**:
- change border radius from 25px to 30px for search bars
- add flex-grow property to search bars
- add styling rule for search button

Did this break anything?
- [x] No

Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)

Checklist:
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Refactor test suites, and fix minor bugs

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -217,7 +227,13 @@ function renderFilteredSavedRecipes() {
       <div class = "recipe-info">
         <p>${recipe.name}</p>
         <button class="delete-recipe-button ${recipe.name}" name="${recipe.name}">🗑️</button>
-      </div>`)
+      </div>`);
+  if (!savedFiltered.length) {
+    savedRecipeDisplay.innerHTML = `
+    <div class="no-recipe-found-message">
+      <p>Sorry, ${currentUser.name}, no recipes match these tags.</p>
```
Comment (andreasorensen) [inline src/domUpdates.js:234]: Love the error handling! Nice job!

**Review by andreasorensen — APPROVED** [review]
Great job with this refactor!

File: src/domUpdates.js [hunk 2]
```diff
@@ -261,6 +277,12 @@ function renderFilteredRecipes() {
       <div class = "recipe-info">
         <p>${recipe.name}</p>
       </div>`)
+    if (!filtered.length) {
+      allRecipeDisplay.innerHTML = `
+      <div class="no-recipe-found-message">
+        <p>Sorry, ${currentUser.name}, no recipes match these tags.</p>
+      </div>`
+    }
```
Comment (fernandorobles97) [inline src/domUpdates.js:285]: Nice job adding error handling here for when there are no recipes that include the checked tags!

**Review by fernandorobles97 — APPROVED** [review]

**Review by hvohr — APPROVED** [review]
this search bar is incredible, nicely done!!!

**Pull request merged**